runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperConfig.java

```diff
@@ -70,11 +70,22 @@ default boolean quickhopOutOfDanger()
 		return true;
 	}
 
+	@ConfigItem(
+		keyName = "quickHopRegionFilter",
+		name = "Quick-hop to specific region only",
```

Comment:
```suggestion
		name = "Quick-hop region",
```

the longer version probably won't fit?